ClosedXML/Excel/AutoFilters/XLFilterColumn.cs

```diff
@@ -179,40 +179,40 @@ public void BelowAverage()
             LessThan(minValue).Or.GreaterThan(maxValue);
         }
 
+        public static Func<String, Object, Boolean> BeginsWithFunction => (value, input) => ((string)input).StartsWith(value, StringComparison.InvariantCultureIgnoreCase);
```

Comment:
If I'm not mistaken, this way it'll produce a new instance of a function on each call. 
`public static Func<String, Object, Boolean> BeginsWithFunction { get; } = (value, input) => ((string)input).StartsWith(value, StringComparison.InvariantCultureIgnoreCase);` — must be cheaper, though I didn't test.

Reply:
This syntax does actually create a property. I confirmed with ILSpy by decompiling to C#3 syntax:
```c#
public static Func<string, object, bool> BeginsWithFunction
{
    get
    {
        return (string value, object input) => ((string)input).StartsWith(value, StringComparison.InvariantCultureIgnoreCase);
    }
}
```

Property:
`public static Func<String, Object, Boolean> BeginsWithFunction => (value, input) => ((string)input).StartsWith(value, StringComparison.InvariantCultureIgnoreCase);`
Variable:
`public static Func<String, Object, Boolean> BeginsWithFunction = (value, input) => ((string)input).StartsWith(value, StringComparison.InvariantCultureIgnoreCase);`

Reply:
Or wait... that still creates a new function for every `get` that is called, right?

Reply:
I think so.